_docs/tasks/security/mutual-tls.md

```diff
@@ -92,7 +92,7 @@ There are several steps:
    
 1. send requests to another service, for example, details.
    ```bash
-   curl https://details:9080 -v --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem --cacert /etc/certs/root-cert.pem -k
+   curl https://productpage:9080 -v --key /etc/certs/key.pem --cert /etc/certs/cert-chain.pem --cacert /etc/certs/root-cert.pem -k
```

Comment:
Currently somehow only productpage works, so this PR is for temporary workaround. I am looking into the problem.

Reply:
that seems wrong - makes it seem like it needs the ingress ?

what happens exactly when using another svc ?